lib/oli_web/live/delivery/remix_section.html.heex

```diff
@@ -28,7 +28,7 @@
             <p>There's nothing here.</p>
           </div>
         <% end %>
-        <%= for {node, index} <- Enum.with_index(@active.children) |> Enum.filter(fn {c, _i} -> c.uuid != @dragging end) do %>
+        <%= for {node, index} <- HierarchyPicker.sort_items(@active.children, assigns) |> Enum.with_index() |> Enum.filter(fn {c, _i} -> c.uuid != @dragging end) do %>
```

Comment:
if possible, I think this would be a great view helper vs doing it inline with the comprehension.

Reply:
Makes sense, will do! Thank you 👍🏼